All In/Features/Profile/ProfileView.swift

```diff
@@ -92,6 +95,35 @@ struct ProfileView: View {
         }
     }
 
+    private var photoPicker: some View {
+        VStack {
+            PhotosPicker(
+                selection: $imageSelection,
+                matching: .images
+            ) {
+                image
+                    .resizable()
+                    .scaledToFill()
+                    .frame(width: 125, height: 125)
+                    .background(.white)
+                    .cornerRadius(100)
```

Comment:
cornerRadius is being deprecated, you want to use:

`
.clipShape(.rect(cornerRadius: 100))
`